src/frontend/src/lib/constants/app.constants.ts

```diff
@@ -17,6 +25,12 @@ export const POH_ISSUER_CANISTER_ID = LOCAL
 			? import.meta.env.VITE_IC_POH_ISSUER_CANISTER_ID
 			: undefined;
 
+export const POH_ISSUER_ORIGIN = nonNullish(POH_ISSUER_CANISTER_ID)
```

Comment:
Should we have `ic0.app` in some constant?

Reply:
Yes, makes sense. Done!